src/collections/programs/lfx-2021/lfx-2021.mdx

```diff
@@ -19,7 +19,7 @@ import lfx_logo from "./Linux_Foundation_logo.svg";
 
 <div>
 <ul>
-  <li><strong>Full-Time Terms:</strong></li>
+  <li class="lfx-timeline_terms"><strong>Full-Time Terms:</strong></li>
```

Comment:
Move this out to be a `h4` and update the `li` below i.e, `Spring Term....` to be the header of the list.

Reply:
Update the same in the other files as well.